SK.Application.UnitTests/Events/Commands/EditEventCommandTest.cs

```diff
@@ -0,0 +1,100 @@
+﻿using AutoMapper;
+using FluentAssertions;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Localization;
+using Moq;
+using NUnit.Framework;
+using SK.Application.Common.Exceptions;
+using SK.Application.Common.Interfaces;
+using SK.Application.Common.Resources.Events;
+using SK.Application.Events.Commands;
+using SK.Application.Events.Commands.EditEvent;
+using SK.Domain.Entities;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace SK.Application.UnitTests.Events.Commands
+{
+    public class EditEventCommandTest
+    {
+        private readonly Guid id;
+        private readonly Mock<DbSet<Event>> dbSetEvent;
+        private readonly Mock<IApplicationDbContext> context;
+        private readonly Mock<IStringLocalizer<EventsResource>> stringLocalizer;
+        private readonly Mock<IMapper> mapper;
+
+        private readonly Event @event;
+        private readonly EventCreateOrEditDto eventDto;
+
+        public EditEventCommandTest()
+        {
+            id = new Guid();
+            dbSetEvent = new Mock<DbSet<Event>>();
+            context = new Mock<IApplicationDbContext>();
+            stringLocalizer = new Mock<IStringLocalizer<EventsResource>>();
+            mapper = new Mock<IMapper>();
+
+            @event = new Event { Id = id };
```

Comment:
Maybe better idea is to name it eventEntity or something similar isntead @event?